.github/workflows/build-docker.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
       - name: Setup variables
         run: |
           # shellcheck disable=SC2129
-          echo "IMAGE_TAG=$(if [ ${{ github.ref_name }} == master ]; then echo latest; elif [ ${{ github.event_name }} == release ]; then echo ${{ github.ref_name }}; else echo staging; fi)" >> "$GITHUB_ENV"
```

Comment:
Unrelated fix I'm sneaking into this PR. We had the wrong default branch name and so weren't creating any `latest` tags.